dev-packages/browser-integration-tests/utils/generatePlugin.ts

```diff
@@ -129,14 +129,17 @@ function generateSentryAlias(): Record<string, string> {
 
   return Object.fromEntries(
     packageNames.map(packageName => {
+      // pluggable integrations exist in the browser package
+      const actPackageName = packageName === 'integrations' ? 'browser' : packageName;
```

Comment:
I think this can be removed again - there is no more package with that name I believe?